detectors/node/opentelemetry-resource-detector-azure/README.md

```diff
@@ -27,48 +27,48 @@ const tracerProvider = new NodeTracerProvider({ resource });
 
 ## Available Detectors
 
+This package implements Semantic Convention [Version 1.19.0](https://github.com/open-telemetry/opentelemetry-specification/blob/v1.7.0/semantic_conventions/README.md).
```

Comment:
```suggestion
This package implements Semantic Convention [Version 1.19.0](https://github.com/open-telemetry/opentelemetry-specification/blob/v1.19.0/semantic_conventions/README.md).
```

Reply:
Doh, I missed updating the link. Take this suggestion. :)